<launch>
  <node name="odometry_simulator" pkg="voxgraph" type="odometry_simulator" output="screen">
    <rosparam file="$(find voxgraph)/config/odometry_simulator.yaml"/>
  </node>

  <node name="voxgraph_mapper" pkg="voxgraph" type="voxgraph_mapping_node" output="screen">
<!--  launch-prefix="gdb -ex run &#45;&#45;args">-->
    <rosparam file="$(find voxgraph)/config/voxgraph_mapper.yaml"/>
  </node>

  <node name="player" pkg="rosbag" type="play" output="screen"
        args="--quiet -s50 /home/victor/data/building_sim/terrain_coverage_2019-01-21-23-26-31.bag"/>

  <node pkg="rviz" type="rviz" name="$(anon rviz)" respawn="false" output="screen"/>
</launch>